src/components/text-input/text-input.scss

```diff
@@ -0,0 +1,24 @@
+// label.text-input {
```

Comment:
Not a fan of commented files being pushed, does this hold some value or it or was it something you missed removing from the PR?

Reply:
Forgot about it after the move to bulma.io - will be removed